src/main/java/core/basesyntax/model/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.model;
+
+public class Circle extends Figure implements FigureRealisation {
```

Comment:
```suggestion
public class Circle extends Figure {
```

if Figure implements FigureRealisation then all it's child classes implements FigureRealisation